src/components/AdminSettings/Federation.vue

```diff
@@ -33,28 +33,126 @@
 			@update:checked="saveFederationEnabled">
 			{{ t('spreed', 'Enable Federation in Talk app') }}
 		</NcCheckboxRadioSwitch>
+
+		<template v-if="isFederationEnabled">
+			<h3>{{ t('spreed', 'Permissions') }}</h3>
+
+			<NcCheckboxRadioSwitch :checked="isFederationIncomingEnabled"
+				:disabled="loading"
+				type="switch"
+				@update:checked="saveFederationIncomingEnabled">
+				{{ t('spreed', 'Allow users to be invited to federated conversations') }}
+			</NcCheckboxRadioSwitch>
+
+			<NcCheckboxRadioSwitch :checked="isFederationOutgoingEnabled"
+				:disabled="loading"
+				type="switch"
+				@update:checked="saveFederationOutgoingEnabled">
+				{{ t('spreed', 'Allow users to invite federated users into conversation') }}
+			</NcCheckboxRadioSwitch>
+
+			<NcCheckboxRadioSwitch :checked="isFederationOnlyTrustedServersEnabled"
+				:disabled="loading"
+				type="switch"
+				@update:checked="saveFederationOnlyTrustedServersEnabled">
+				{{ t('spreed', 'Allow only federation with trusted servers') }}
```

Comment:
```suggestion
				{{ t('spreed', 'Allow federation with trusted servers only') }}
```
🤔

Reply:
Or maybe even 
```suggestion
				{{ t('spreed', 'Only allow federation with trusted servers') }}
```

Reply:
Also in the option to enable federation it’s „Federation“ with a capital „F“. Guess we should keep it consistent either way

Reply:
I though of `Federation` as a feature in general, and `federation` as a single conversation.

Maybe `Only allow to federate with trusted servers` would be better here?